README.md

```diff
@@ -156,6 +156,13 @@ Tide's git capabilities are inherited from fish's built-in [fish_git_prompt][].
 | virtual_env_icon    | icon to display in front of virtual_env item                                                                                           | ''         |
 | virtual_env_display | Options are venvName and projectName. venvName is the virtual env directory, most often .venv. projectName is the directory above that | projectName |
 
+### nvm
+
+| Variable        | Description                                                                                                                    | Default     |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------ | ----------- |
+| nvm_env_color   | color of nvm item                                                                                                              | 00AFAF      |
```

Comment:
These should begin with `nvm` not `nvm_env` as the item is call nvm, and the header for the section is called nvm